### PR TITLE
Aruha 1099 reached max retries: 4

### PR DESCRIPTION
**Pull request by antban**
[ARUHA-1099: (reached max retries: 4)]

## Description
Use exponential backoff with different max retries for AWS communication

## Review
- [ ] Tests
- [ ] Documentation
- [ ] CHANGELOG

## Deployment Notes
There are no deployment notes, cause changed part is related only to aws deployment scripts, and not to service.

### CODE REVIEW

File: instance_control/bubuku_cluster.py [hunk 1]
```diff
@@ -37,13 +37,13 @@ def attach(volume_id: str, cluster_config: str):
...
 @cli.command('create', help='Launch instance with EBS attached')
-@click.option('--cluster-size', type=int)
+@click.option('--cluster-config', default='bubuku-1.json')
 @click.option('--availability-zone', default=None)
 @click.option('--image-version', help='Docker image version to use. By default the version from config is used. '
                                       'If provided then verrides image version from config')
-@click.option('--cluster-config', default='bubuku-1.json')
-def create(cluster_size: int, availability_zone: str, image_version: str, cluster_config: str):
-    CreateCommand(cluster_config, cluster_size, availability_zone, image_version).run()
+@click.option('--count', default=1, type=int, help='Count of instances to create', show_default=True)
```
Comment (adyach) [inline instance_control/bubuku_cluster.py]: --instance-count

File: instance_control/aws/ec2_node.py [hunk 2]
```diff
@@ -4,153 +4,157 @@
...
+                    subnet_,
+                    cluster_config['taupage_amis'],
+                    security_group_id=cluster_config['security_group']['GroupId'],
+                    cluster_config=cluster_config))
+        # wait for all instances to start
+        while starting_instances:
+            _LOG.info("Waiting for instances to start: {}".format(starting_instances))
+            time.sleep(5)
+            resp = self.aws.ec2_client.describe_instances(InstanceIds=starting_instances)
+            started = []
```
Comment (adyach) [inline instance_control/aws/ec2_node.py]: started_instances vs started

**Comment by antban** [comment]
@adyach Renamed: https://github.com/zalando-nakadi/bubuku/pull/106/commits/1a16f9224be38521cababfde7e22ee7ff231e74d

**Comment by adyach** [comment]
👍

**Comment by antban** [comment]
:+1:

**Comment by antban** [comment]
:+1:

**Comment by adyach** [comment]
👍

**Pull request merged**